dd-sdk-android/apiSurface

```diff
@@ -59,6 +63,22 @@ class com.datadog.android.log.Logger
   fun addTag(String)
   fun removeTag(String)
   fun removeTagsWithKey(String)
+object com.datadog.android.rum.GlobalRum
```

Comment:
As I understand, `GlobalRum` is just a helper to register `RumMonitor` instance retrieved with its builder. This is kinda different than what we offer for `Logger` and `Tracer` (no global helpers). Why is that? Do we plan to update `Logger` and `Tracer` to this pattern or we want to keep the API different for that features?

Reply:
Actually this is inspired by the GlobalTracer (part of the opentracing lib). 
We do that because we need a single instance to keep track of the active views.
It makes sense to have Loggers separated as they can be independant and have different attributes / tags linked. But Rum and Tracer look at the app execution on a wider scope, and needs to use the same instance everywhere.